dockerfiles/flow-libs-dev-rel/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG BUILD_TYPE=release
 COPY install_lib.sh /usr/bin/install_lib.sh
 
 # install libraries
-RUN /usr/bin/install_lib.sh petsc       3.6.1 ${BUILD_TYPE}
+RUN /usr/bin/install_lib.sh petsc       3.8.3 ${BUILD_TYPE}
 RUN /usr/bin/install_lib.sh yamlcpp     0.5.2 ${BUILD_TYPE}
-RUN /usr/bin/install_lib.sh armadillo   3.4.3 ${BUILD_TYPE}
+RUN /usr/bin/install_lib.sh armadillo   8.4.3 ${BUILD_TYPE}
```

Comment:
Updating libs is just a first step the second must be to make flow123d to work with them. Is it possible to use different images for different branches of Flow123d? Is it enough to set .gitmodules to different commits of docker-config repository?